### PR TITLE
Features/15 dashboard eventpanel admin abdi

### PR DESCRIPTION
**Pull request by abwork**

### CODE REVIEW

**Comment by trungh13** [comment]
first resolve the conflict
2nd, admin ?

File: src/App.css [hunk 1]
```diff
@@ -33,21 +33,24 @@
 }
 
 .bold{
-  font-weight: 600;
+  font-weight: 700;
```
[the remark this answers is not in the document]
Comment (abwork) [reply src/App.css]: This isn't random. 600 was too light for my titles. I will define a new style. Can we also keep our styles independent as much as possible?

File: src/containers/Dashboard/components/EventPanel/index.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  letter-spacing: -0.05em;
+  padding-top: 0.8em;
+  color: var(--secondary);
+}
+
+.row .col {
+  padding: 0;
+}
+
+.row {
```
[the remark this answers is not in the document]
Comment (abwork) [reply src/containers/Dashboard/components/EventPanel/index.css]: I will define a new style just for event panel.

**Comment by abwork** [comment]
I am working on this branch as of yesterday. That's why I pushed it here. It seems that we are still doing Sprint I. The styling was so basic just to get the MVP ready. You guys are very focused and interested in that 💃 sadly, I hadn't any guidance on the style. My panel was empty from the designs  saw.

**Pull request closed without merging**